shell/browser/api/electron_api_session.cc

```diff
@@ -1058,17 +1059,20 @@ void Session::SetSpellCheckerLanguages(
   // Enable spellcheck if > 0 languages, disable if no languages set
   browser_context_->prefs()->SetBoolean(spellcheck::prefs::kSpellCheckEnable,
                                         !languages.empty());
+#endif
 }
 
 void SetSpellCheckerDictionaryDownloadURL(gin_helper::ErrorThrower thrower,
                                           const GURL& url) {
+#if !BUILDFLAG(IS_MAC)
```

Comment:
_(pedantic)_ arguably a `semver/minor` change since this an invalid URL no longer throws on macOS?

Reply:
i'd say potentially, but given that the only possible behavioral effect of this API before this change would have been to error incorrectly, informing users that an incorrect behavior is now a correct behavior is likely not something we'd want to gate behind semver-minor mechanics 🤔